app/jobs/change_pickup_library_job.rb

```diff
@@ -0,0 +1,16 @@
+class ChangePickupLibraryJob < ApplicationJob
+  queue_as :default
+
+  def perform(pass_these)
+    symphony_client = SymphonyClient.new
```

Comment:
Can we also pass `symphony_client` with `pass_these`? So there won't be need for this line `symphony_client = SymphonyClient.new` here. Would it make sense to pass it? 🤔

Reply:
Yup!

Reply:
Tried to do this and got a "Serialization" error. Apparently you can't pass certain objects to an activejob?